packages/suite/src/storage/migrations/index.ts

```diff
@@ -885,4 +885,110 @@ export const migrate: OnUpgradeFunc<SuiteDBSchema> = async (
+                suiteSettings.evmSettings.confirmExplanationModalClosed?.matic;
+
+            suiteSettings.evmSettings.explanationBannerClosed.pol =
+                // @ts-expect-error
+                suiteSettings.evmSettings.explanationBannerClosed?.matic;
+
+            // @ts-expect-error
+            delete suiteSettings.evmSettings.explanationBannerClosed?.matic;
+            // @ts-expect-error
+            delete suiteSettings.evmSettings.confirmExplanationModalClosed?.matic;
+
+            return suiteSettings;
+        });
+
+        const backendSettings = transaction.objectStore('backendSettings');
+        // @ts-expect-error
+        const maticBackendSettings = await backendSettings.get('matic');
+        if (maticBackendSettings) {
+            backendSettings.add(maticBackendSettings, 'pol');
+            // @ts-expect-error
+            backendSettings.delete('matic');
+        }
+
+        const tokenManagement = transaction.objectStore('tokenManagement');
+        const maticTokenManagementShow = await tokenManagement.get('matic-coin-show');
+        const maticTokenManagementHide = await tokenManagement.get('matic-coin-hide');
+        if (maticTokenManagementShow && maticTokenManagementHide) {
+            tokenManagement.add(maticTokenManagementShow, 'pol-coin-show');
+            tokenManagement.add(maticTokenManagementHide, 'pol-coin-hide');
+
+            tokenManagement.delete('matic-coin-show');
+            tokenManagement.delete('matic-coin-hide');
+        }
+
+        // matic removed watch-only wallet data
+        await updateAll(transaction, 'discovery', discovery => {
+            discovery.networks = discovery.networks.filter(
+                // @ts-expect-error
+                network => network !== 'matic',
+            );
+
+            discovery.failed = [];
+
+            return discovery;
+        });
+
+        await updateAll(transaction, 'txs', tx => {
+            // @ts-expect-error
+            if (tx.tx.symbol === 'matic') {
+                return null;
+            }
+
+            return tx;
+        });
+
+        await updateAll(transaction, 'accounts', account => {
+            // @ts-expect-error
+            if (account.symbol === 'matic') {
+                return null;
+            }
+
+            return account;
+        });
+
+        await updateAll(transaction, 'graph', graph => {
+            // @ts-expect-error
+            if (graph.account.symbol === 'matic') {
+                return null;
+            }
+
+            return graph;
+        });
+
+        const historicRates = transaction.objectStore('historicRates');
+        const historicRatesKeys = await historicRates.getAllKeys();
+        const historicRatesKeysWithMatic = historicRatesKeys.filter(key => key.includes('matic'));
+
+        historicRatesKeysWithMatic.forEach(key => {
+            historicRates.delete(key);
+        });
+
+        const sendFormDrafts = transaction.objectStore('sendFormDrafts');
```

Comment:
`formDrafts` (probably trade section?) not migrated or deleted.

Reply:
🤯